google/cloud/storage/internal/grpc_notification_request_parser.cc

```diff
@@ -21,6 +21,14 @@ namespace storage {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace internal {
 
+google::storage::v2::GetNotificationRequest ToProto(
+    GetNotificationRequest const& rhs) {
+  google::storage::v2::GetNotificationRequest request;
+  request.set_name("projects/_/buckets/" + rhs.bucket_name() +
```

Comment:
just curious, is there ever a situation where we would prefer to specify the project instead of using `_`?